rpc/class_test.go

```diff
@@ -218,3 +218,23 @@ func TestClassHashAt(t *testing.T) {
 		assert.Equal(t, expectedClassHash, classHash)
 	})
 }
+
+func TestJunoGetNodesFromRoot(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	t.Cleanup(mockCtrl.Finish)
+
+	mockReader := mocks.NewMockReader(mockCtrl)
+	log := utils.NewNopZapLogger()
+	handler := rpc.New(mockReader, nil, nil, "", log)
+
+	t.Run("Empty blockchain", func(t *testing.T) {
```

Comment:
It would be nice to have a test with a non-empty blockchain, but otherwise nice work

Reply:
Added a new test with non-empty blockchain